pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -942,5 +979,23 @@ func GetBootstrapClusterRoleBindings() []authorizationapi.ClusterRoleBinding {
 				{Kind: authorizationapi.SystemGroupKind, Name: UnauthenticatedGroup},
 			},
 		},
+
+		// Grant all build strategy roles by default
+		// Cluster admins can remove these role bindings if they don't want them
```

Comment:
add something like "and its "safe" because `system:authenticated` is excluded from clusterrolebinding reconciliation by default."